site/content/en/docs/reference/labels-and-annotations.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Labels and Annotations"
+linkTitle: "Labels and Annotations"
+date: 2024-10-09
+---
+
+This page serves as a reference for all labels and annotations in Kueue.
```

Comment:
Can we have these labels/annotations sorted?

Reply:
I sorted them by most used. Which order do you prefer?

Reply:
In general, users do not have a context which labels and annotations are mostly used.
So, alphabetical order would be better, IMO.

Reply:
> In general, users do not have a context which labels and annotations are mostly used. So, alphabetical order would be better, IMO.

Make sense.

Reply:
Done